{"name": "Login", "status": "failed", "statusDetails": {"message": "Failed: test is failed", "trace": "self = <selenium_automation.test_allure_reports.Test_Allure_Reports object at 0x0000022C05385B50>\n\n    @allure.title(\"Login\")\n    @allure.description(\"make login with given password and male\")\n    def test_make_report(self):\n        try:\n            self.insert_mail(\"admin@yourstore.com\")\n            self.insert_password(\"admin\")\n            self.click_login()\n>           self.verify_login()\n\nselenium_automation\\test_allure_reports.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\na = (<selenium_automation.test_allure_reports.Test_Allure_Reports object at 0x0000022C05385B50>,), kw = {}, __tracebackhide__ = True, params = OrderedDict()\nargs = ['<selenium_automation.test_allure_reports.Test_Allure_Reports object at 0x0000022C05385B50>']\n\n    @wraps(func)\n    def impl(*a, **kw):\n        __tracebackhide__ = True\n        params = func_parameters(func, *a, **kw)\n        args = list(map(lambda x: represent(x), a))\n        with StepContext(self.title.format(*args, **params), params):\n>           return func(*a, **kw)\n\nvenv\\Lib\\site-packages\\allure_commons\\_allure.py:192: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium_automation.test_allure_reports.Test_Allure_Reports object at 0x0000022C05385B50>\n\n    @allure.step(\"verify if current url is wright\")\n    def verify_login(self):\n        title = \"https://admin-demo.nopcommerce.com/admi/\"\n>       assert driver.current_url == title\nE       AssertionError: assert 'https://admi...ce.com/admin/' == 'https://admi...rce.com/admi/'\nE         - https://admin-demo.nopcommerce.com/admi/\nE         + https://admin-demo.nopcommerce.com/admin/\nE         ?                                        +\n\nselenium_automation\\test_allure_reports.py:47: AssertionError\n\nDuring handling of the above exception, another exception occurred:\n\nself = <selenium_automation.test_allure_reports.Test_Allure_Reports object at 0x0000022C05385B50>\n\n    @allure.title(\"Login\")\n    @allure.description(\"make login with given password and male\")\n    def test_make_report(self):\n        try:\n            self.insert_mail(\"admin@yourstore.com\")\n            self.insert_password(\"admin\")\n            self.click_login()\n            self.verify_login()\n        except:\n            self.attach_file()\n>           pytest.fail(\"test is failed\")\nE           Failed: test is failed\n\nselenium_automation\\test_allure_reports.py:25: Failed"}, "description": "make login with given password and male", "steps": [{"name": "insert email", "status": "passed", "parameters": [{"name": "mail", "value": "'admin@yourstore.com'"}], "start": 1680713571310, "stop": 1680713571467}, {"name": "insert given pas", "status": "passed", "parameters": [{"name": "password", "value": "'admin'"}], "start": 1680713571468, "stop": 1680713571594}, {"name": "click on button", "status": "passed", "start": 1680713571594, "stop": 1680713574931}, {"name": "verify if current url is wright", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'https://admi...ce.com/admin/' == 'https://admi...rce.com/admi/'\n  - https://admin-demo.nopcommerce.com/admi/\n  + https://admin-demo.nopcommerce.com/admin/\n  ?                                        +\n", "trace": "  File \"C:\\Users\\philip itkin\\PycharmProjects\\pythonProject\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\philip itkin\\PycharmProjects\\pythonProject\\selenium_automation\\test_allure_reports.py\", line 47, in verify_login\n    assert driver.current_url == title\n"}, "start": 1680713574931, "stop": 1680713574973}, {"name": "attach sreenshot of fail", "status": "passed", "attachments": [{"source": "fc1e3234-c8b0-4a83-95fc-45577492b7c8-attachment.png", "type": "image/png"}], "start": 1680713574998, "stop": 1680713575124}], "start": 1680713571310, "stop": 1680713575124, "uuid": "5ef14f95-8acf-4dfb-8a81-003c03af71e3", "historyId": "c0a8480581af5973db44eca4304dd109", "testCaseId": "45ae8a157a851f3c0997ec1082d3c50a", "fullName": "selenium_automation.test_allure_reports.Test_Allure_Reports#test_make_report", "labels": [{"name": "parentSuite", "value": "selenium_automation"}, {"name": "suite", "value": "test_allure_reports"}, {"name": "subSuite", "value": "Test_Allure_Reports"}, {"name": "host", "value": "DESKTOP-T6F5IU8"}, {"name": "thread", "value": "15572-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "selenium_automation.test_allure_reports"}]}